fix_imports.sh

```
@@ -0,0 +1,13 @@
+for file in \
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)